server/plugin.go

```diff
@@ -204,6 +208,91 @@ func (p *Plugin) OnActivate() error {
 	}
 
 	go p.initStats()
+
+	go func() {
+		time.Sleep(time.Second * 10)
+
+		instances, err := p.instanceStore.LoadKnownJIRAInstances()
+		if err != nil {
+			p.API.LogError("unable to register autolinks", "err", err)
+			return
+		}
+
+		for url := range instances {
+			instance, err := p.instanceStore.LoadJIRAInstance(url)
+			if err != nil {
+				continue
+			}
+
+			jci, ok := instance.(*jiraCloudInstance)
+			if !ok {
+				p.API.LogWarn("only cloud instances supported for autolink", "err", err)
+				continue
+			}
+
+			if err := p.InstallAutolinkForCloudInstance(jci); err != nil {
```

Comment:
1/5 s/Install/Configure/; 0/5 s/InstallAutolinkForCloudInstance/configureCloud/

OT: I started thinking using this pattern more, to improve the code readability:

```go
type autolink struct {}

func (a autolink) Configure(...link.Link) {...}

...
al := NewAutolink()
err = al.Configure(ji.Autolinks()...)
```

Reply:
It seems like an "Install" to me since we are sending data to a separate system?

> reading a function name, one should need to switch to a "parsing" mode, names should read naturally, with the emphasis on the client (caller)

+1000 You read code way more then you write it.

> 1/5 we should rename Link to Autolink

Ya I struggled with the name there. I think you are right it should be Autolink. Will change.

Reply:
`autolink.Install(...*autolink.Autolink)` wfm

Reply:
0/5 To me, install has the connotation of "install and enable the plugin as needed" which is not true, but I can see your POV as well.

Reply:
@levb Switched to using a new autolink client to address this feedback.